src/Microsoft.Data.SqlClient/netcore/src/Properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,4 @@
-﻿using System;
-using System.Runtime.CompilerServices;
-using System.Reflection;
-using System.Security;
+﻿using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.Data.SqlClient.NetCore.UnitTests")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
What is that assembly?

Reply:
Needed to expose internals to the dependencies of Moq.